src/main/java/org/folio/event/util/KafkaEventUtil.java

```diff
@@ -27,4 +28,9 @@ public static String extractValueFromHeaders(List<KafkaHeader> headers, String k
       .orElse(null);
   }
 
+  public static Map<String, String> getHeaderMap(List<KafkaHeader> headers) {
+    var caseInsensitiveMap = new TreeMap<String, String>(String.CASE_INSENSITIVE_ORDER);
+    headers.forEach(header -> caseInsensitiveMap.put(header.key(), header.value().toString()));
```

Comment:
1. in case of header.value() is null 
headers.forEach(header -> {
        var key = header.key();
        var value = header.value();
        caseInsensitiveMap.put(key, value != null ? value.toString() : "");
    });
    2. We can also use this option
     var headers = new CaseInsensitiveMap<>(kafkaHeadersToMap(kafkaConsumerRecord.headers())); example:https://github.com/folio-org/mod-orders-storage/blob/d14a5447ca8e1f38d24c3cd4df9b20b2231dc13e/src/main/java/org/folio/event/handler/InventoryCreateAsyncRecordHandler.java#L70
     
     kafkaHeadersToMap(existing method in dependency)
     because TreeMap have to main order and access to field is o(logn) while CaseInsentivemap is litte more performance for get operation.